config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :mechanics, only: [:show, :update] do
```

Comment:
Not sure we need an `update` action for these stories - we do need some kind of `create` and I see it is nested under this resource. You can also have an entirely separate line for mechanics_ride, like so: 
```
resources :mechanics_ride, only: [:create]
```

In addition, you can also hand-roll a single route like this if it doesn't quite fit into your other blocks. There is often a mix of both `resources` and hand-rolled, which can be fine.

Reply:
So I added the update because my action was update because I felt like the adding a ride was rather updating the mechanics workload as opposed to creating but I will have to double check the controller!

Reply:
Ah I see now - in that case, I would say that _updating_ a mechanic would mean we're updating a mechanic's name or their years of experience, that's all that they hold in their table. 
What we're really doing in US2 is _creating_ a relationship between a mechanic and a ride. That's why I was originally suggesting a create action might make more sense.